opencti-platform/opencti-front/src/private/components/settings/users/User.tsx

```diff
@@ -35,7 +35,7 @@ import Transition from '../../../../components/Transition';
 import { User_user$key } from './__generated__/User_user.graphql';
 import Chart from '../../common/charts/Chart';
 import { UserSessionKillMutation } from './__generated__/UserSessionKillMutation.graphql';
-import { UserUserSessionsKillMutation } from './__generated__/UserUserSessionsKillMutation.graphql';
+import { UserSessionsKillMutation } from './__generated__/UserUserSessionsKillMutation.graphql';
```

Comment:
build is on error : 

`src/private/components/settings/users/User.tsx(38,42): error TS2307: Cannot find module './__generated__/UserUserSessionsKillMutation.graphql' or its corresponding type declarations.`